build/custom.mk

```diff
@@ -14,6 +14,7 @@ BUILD_HASH_SHORT = $(shell git rev-parse --short HEAD)
 LDFLAGS += -X "main.BuildDate=$(BUILD_DATE)"
 LDFLAGS += -X "main.BuildHash=$(BUILD_HASH)"
 LDFLAGS += -X "main.BuildHashShort=$(BUILD_HASH_SHORT)"
+LDFLAGS += -X "main.Provider=$(PROVIDER)"
```

Comment:
I wonder if we want to be more specific with `CALENDAR_PROVIDER` instead. There isn't another concept of "Provider" in the plugin project, though probably better for a new dev when reading the code